apps/web/pages/cancel/[uid].tsx

```diff
@@ -272,10 +278,11 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
   });
 
   let recurringInstances = null;
-  if (booking.eventType?.recurringEvent) {
+  if (booking.eventType?.recurringEvent && allRemainingBookings === "true") {
     recurringInstances = await prisma.booking.findMany({
       where: {
         recurringEventId: booking.recurringEventId,
+        NOT: [{ status: "CANCELLED" }, { status: "REJECTED" }],
```

Comment:
Now we show the correct recurring instances remaining when trying to cancel the entire series.